packages/components/table-v2/src/use-table.ts

```diff
@@ -84,24 +84,37 @@ function useTable(props: TableV2Props) {
     resetAfterIndex,
   })
 
+  const rowsHeight = computed(() => {
+    const { estimatedRowHeight, rowHeight } = props
+    const _data = unref(data)
+    if (isNumber(estimatedRowHeight)) {
```

Comment:
`estimatedRowHeight` seems to be useless below, can it be removed?

Reply:
estimatedRowHeight is used to determine whether the row heights are dynamic. rowHeights only have values when estimatedRowHeight is set to a specific number. When estimatedRowHeight is undefined, it's necessary to use _data.length * rowHeight to calculate the total height of all rows. It appears that estimatedRowHeight cannot be removed.

Reply:
OK, it looks fine to me. But it still needs to be reviewed by others, please be patient.